### PR TITLE
Fixed small typo

### PR DESCRIPTION
**Pull request by AlwaysLivid**

### CODE REVIEW

File: src/instagram.ts [hunk 1]
```diff
@@ -148,7 +148,7 @@ export class Instagram {
 			log.warn("Failed to connect to client", err);
 			await this.puppet.sendStatusMessage(puppetId, err);
 			await this.puppet.sendStatusMessage(puppetId, `**disconnected!** You have been logged out!` +
-				` Please use \`relink ${puppetId} <username> <password\` to log in again!`);
+				` Please use \`relink ${puppetId} <username> <password>` to log in again!`);
```
Comment (Sorunome) [inline src/instagram.ts:151]: the `\`` still needs escaping, so `>\\\``

Comment (AlwaysLivid) [reply src/instagram.ts:151]: Oops, sorry!

Comment (AlwaysLivid) [reply src/instagram.ts]: There we go, sorry about that. Feel free to squash. @Sorunome

**Comment by Sorunome** [comment]
Well, thanks for the PR!

**Comment by AlwaysLivid** [comment]
> Well, thanks for the PR!

Eh, that really was small change but it did catch my eye as I was using the bridge myself, so why not? :D

**Pull request merged**